luatest/runner.lua

```diff
@@ -148,6 +149,13 @@ function Runner.parse_cmd_line(args)
             result.output_file_name = next_arg()
         elseif arg == '--repeat' or arg == '-r' then
             result.exe_repeat = tonumber(next_arg()) or error('Invalid value for -r option. Integer required.')
+        elseif arg == '--group-repeat' or arg == '-g' then
+            result.exe_group_repeat = tonumber(next_arg())
+            if result.exe_group_repeat == nil
+            or result.exe_group_repeat < 0
+            then
+                error('Invalid value for -g option. Positive integer required.')
```

Comment:
You can use `arg` instead of '-g' here. It will cover `--repeat-group` option as well.